src/main/resources/db/migration/V1_15__add_prison_contracted_flag.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE prison ADD COLUMN contracted BOOLEAN NOT NULL DEFAULT false;
+
+UPDATE prison SET contracted = true WHERE prison_id IN ('ACI', 'ASI', 'BZI', 'DNI', 'DGI', 'FWI', 'FBI', 'LGI', 'NLI', 'OWI', 'PBI', 'PFI', 'PRI', 'RHI', 'TSI');
```

Comment:
`PYI`, "Parc A" is also a contracted prison

Reply:
Hi, we don't have Parc A in the Prison Register at the moment. I'm raising a separate ticket to get that one added

Reply:
it does show in the [prod site](https://prison-register.hmpps.service.justice.gov.uk/prisons/id/PYI), but simply named as "Parc (HMYOI)"

Reply:
OK, Parc was on the ticket too, it has ID: PRI

Reply:
actually, there are 2 Parc prisons (it was split earlier this year):

https://github.com/ministryofjustice/prison-register/blob/main/src/main/resources/db/migration/V1_1__insert_prisons.sql#L100

https://github.com/ministryofjustice/prison-register/blob/main/src/main/resources/db/migration/V1_7__add_hmp_parc.sql#L1

Reply:
Ah, sorry OK - will update now - was thrown off by the name Parc A